example/swagger-files/parameter-styles-cookie.json

```diff
@@ -0,0 +1,131 @@
+{
+    "openapi": "3.0.0",
+    "info": {
+      "description": "This oas file demonstrates all of the different cookie parameter styles",
+      "version": "1.0.0",
+      "title": "Parameter Styles",
+      "contact": {
+        "email": "aaron@readme.io"
+      },
+      "license": {
+        "name": "Apache 2.0",
+        "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+      }
+    },
+    "paths": {
+        "/nonExplodedFormCookies": {
+            "get": {
+                "summary": "Demonstrate all non-exploded form cookie styles",
+                "operationId": "demonstrateCookies",
```

Comment:
Can you make the operationId's in these definitions? There's some DOM warnings being thrown because `id` attributes are getting duplicated.

![Screen Shot 2021-01-06 at 10 26 59 AM](https://user-images.githubusercontent.com/33762/103806399-cf6eb980-5009-11eb-80d1-104f4b6a384e.png)

Reply:
fixed locally